kueue-addon/deploy/kustomization.yaml

```
@@ -0,0 +1,11 @@
+resources:
+- resources/addon-template.yaml
+- resources/cluster-management-addon.yaml
+- resources/placement.yaml
+
+images:
+- name: kueue-addon-image
+  newName: quay.io/haoqing/kueue-addon
```

```suggestion
  newName: quay.io/open-cluster-management/kueue-addon
```

minor nit as above comment